scene/resources/concave_polygon_shape.h

```diff
@@ -61,12 +61,15 @@ class ConcavePolygonShape : public Shape {
 	static void _bind_methods();
 
 	virtual void _update_shape();
+
 	virtual Vector<Vector3> _gen_debug_mesh_lines();
 
 public:
 	void set_faces(const PoolVector<Vector3> &p_faces);
 	PoolVector<Vector3> get_faces() const;
 
+	Vector<Vector3> get_debug_mesh_lines();
```

Comment:
We could change `_gen_debug_mesh_lines` to `get_debug_mesh_lines` and make it a public method in `Shape`. This way all the different shapes get the same functionality and the code is a bit cleaner.

The rest of the PR looks great :)